week1/homework/src/server.js

```diff
@@ -5,13 +5,33 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+function  getResult(status, response, result){
```

Comment:
the name getResult is a bit odd
Since this is not getting the result from somewhere
rather it is making/building response